src/core/loop_scc_attributes/src/PeriodicVariableSCC.cpp

```diff
@@ -40,6 +41,18 @@ PeriodicVariableSCC::PeriodicVariableSCC(
     period{ per },
     step{ st } {
 
+  // DDLOTT: the algorithm for determining the accumulator in
```

Comment:
This comment is confusing. The class SingleAccumulatorRecomputableSCC is meant to handle a single-accumulator SCC as the name suggests. So it's strange to have a comment in this class saying the algorithm included within is not sufficient for 2-phis (hence multiple accumulators).